README.md

```diff
@@ -181,62 +181,77 @@ All of the below examples use the catalog in mainnet, you may replace the import
 
 **Example 1 - Retrieve all NFT collections on the catalog**
 
-```swift
+```cadence
 import NFTCatalog from 0x49a7cda3a1eecc29
 
 /*
-	The catalog is returned as a `String: NFTCatalogMetadata`
-	The key string is intended to be a unique identifier for a specific collection.
-	The NFTCatalogMetadata contains collection-level views corresponding to each
-	collection identifier.
+    The catalog is returned as a `String: NFTCatalogMetadata`
+    The key string is intended to be a unique identifier for a specific collection.
+    The NFTCatalogMetadata contains collection-level views corresponding to each
+    collection identifier.
 */
-pub fun main(): {String : NFTCatalog.NFTCatalogMetadata} {
-    return NFTCatalog.getCatalog()
+pub fun main(): {String: NFTCatalog.NFTCatalogMetadata} {
+    let catalog = NFTCatalog.getCatalog()
+    let keys = catalog.keys.slice(from: 0, upTo: 10)
```

Comment:
will this not change the sematics of this script? in this new version it will only return the first 10 keys from the catalog?

Reply:
As it was, the script didn't even run, due to some storage capacity error :sweat_smile: It's not possible to return the entire catalog. Now the script is functional at least, and the developer has the chance to view the structure of the response, even partially.

Reply:
Fair point. Maybe change the comment then?

Reply:
Sure thing :+1:

Reply:
Updated in https://github.com/dapperlabs/nft-catalog/pull/106/commits/351480107c9c80b83868cc3c11aa0bcdd8411cd5